lca-cli/ops/ops.go

```diff
@@ -281,6 +269,22 @@ func (o *ops) UnmountAndRemoveImage(img string) error {
 	return nil
 }
 
+func (o *ops) UnmountAndRemoveImage(img string) error {
+	mounted, err := o.IsImageMounted(img)
+	if err != nil {
+		return fmt.Errorf("failed to check if image is mounted: %w", err)
+	}
+	if !mounted {
+		return o.removeImage(img)
+	}
+	if _, err := o.hostCommandsExecutor.Execute(
+		"podman", "image", "umount", img,
+	); err != nil {
+		return fmt.Errorf("failed to unmount image: %w", err)
+	}
+	return o.removeImage(img)
```

Comment:
Would this be equivalent?

```suggestion
	mounted, err := o.IsImageMounted(img)
	if err != nil {
		return fmt.Errorf("failed to check if image is mounted: %w", err)
	}

	if mounted {
		if _, err := o.hostCommandsExecutor.Execute(
			"podman", "image", "umount", img,
		); err != nil {
			return fmt.Errorf("failed to unmount image: %w", err)
		}
	}

	return o.removeImage(img)
```